addons/website_hr_recruitment/models/hr_job.py

```diff
@@ -15,6 +15,7 @@ class Job(models.Model):
         'website.published.multi.mixin',
         'website.searchable.mixin',
     ]
+    _order = "published_date desc nulls last, create_date desc, no_of_recruitment desc, no_of_hired_employee asc"
```

Comment:
Is it needed to have so many orders ? 

I mean you won't have 2 jobs created on the same create_date (as it is a datetime).

```suggestion
    _order = "published_date DESC NULLS LAST, create_date DESC, id"
```